portal/account/register.php

```diff
@@ -0,0 +1,556 @@
+<?php
+//For some reason linked session wants to stick around. Think I have it covered?
+unset($_SESSION['itsme']);
+session_destroy();
+session_start();
+session_regenerate_id(true);
+
+$_SESSION['patient_portal_onsite_two'] = true;
+$_SESSION['authUser'] = 'portal-user';
+$_SESSION['pid'] = true;
+$_SESSION['site_id'] = 'default';
+$_SESSION['register'] = true;
+
+$landingpage = "index.php?site=" . $_SESSION['site_id'];
```

Comment:
Hasn't this session variable been obliterated via above?
(I do know how this code is working, but could collect it as a parameter, and/or save it from the session if that exists before wiping the session)

Reply:
Yep your're right. I forgot to remove the session destroy and carry forward site from portal log in. I was tracking down an issue with sessions. I was tired.